modules/util/css-builder/src/main/java/com/liferay/css/builder/CSSBuilder.java

```diff
@@ -170,11 +142,15 @@ public void close() throws Exception {
 		_sassCompiler.close();
 	}
 
-	public void execute(List<String> dirNames) throws Exception {
+	public void execute() throws Exception {
 		List<String> fileNames = new ArrayList<>();
 
-		for (String dirName : dirNames) {
-			_collectSassFiles(fileNames, dirName, _docrootDirName);
+		String docrootDirName = _cssBuilderArgs.getDocrootDir().toString();
+
+		for (String dirName : _cssBuilderArgs.getDirNames()) {
+			List<String> sassFiles = _collectSassFiles(dirName, docrootDirName);
```

Comment:
If the variable is a `String`, its name should be `*FileName` or `*DirName`. If it's a `File`, its name should be `*File` or `*Dir`.